tests/Fixtures/App/Kernel.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Doctrine\Tests\Fixtures\App;
```

Comment:
Maybe we could respect the same directory / namespace structure than other bundles, where the `App\` NS is directly under `Tests\` ([ref](https://github.com/sonata-project/SonataAdminBundle/blob/604aaccf2a5b3fff4746f12d803bdd6c4fe9e76e/tests/App/AppKernel.php#L14)).